features/raadpleeg-bewoning-op-peildatum/test/bewoners-volgorde.feature

```diff
@@ -0,0 +1,442 @@
+# language: nl
+@api
+Functionaliteit: Sorteren van bewoners op datum aanvang adreshouding, geslachtsnaam, voornamen en geboortedatum
+    De sortering gebeurt op basis van de volgende criteria, in deze volgorde:
```

Comment:
ik zie in verschillende feature files dat deze tekst blok wordt herhaald. In code kennen we de principe Don't Repeat Yourself. Is dit niet iets wat wij ook moeten toepassen in de feature files? Om te voorkomen dat we straks op meerdere plekken dezelfde requirements beschrijvingen moeten aanpassen?